src/com/oltpbenchmark/Results.java

```diff
@@ -171,14 +162,8 @@ public void writeAllCSVAbsoluteTiming(List<TransactionType> activeTXTypes, Print
         for (Sample s : latencySamples) {
```

Comment:
You'll need to change the header above too to only include `Transaction Name, Latency (microseconds)`

Reply:
Done.

Reply:
Done.